### PR TITLE
Give course title input field more space

### LINKED ISSUE #1772: Improve Course Title Editing Box
image

related to https://github.com/ilios/common/issues/1770

image

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #1772 

Selection_046

### CODE REVIEW

File: addon/components/course-header.hbs [hunk 1]
```diff
@@ -26,16 +26,16 @@
         {{/each}}
       </EditableField>
     {{else}}
-      <h2>
+      <h4>
```
Comment (jrjohnson) [inline addon/components/course-header.hbs:29]: I know we don't enforce it everywhere, but the header level here *should* be 2 as it is the critical data on the page.

**Review by jrjohnson — CHANGES_REQUESTED** [review]
Not sure about this solution, why would the title area only be bigger when editing, shouldn't it take up the same space as the full title did because it contains the same characters?

**Review by dartajax — APPROVED** [review]
It works for me.

**Comment by dartajax** [comment]
Sao @jrjohnson's issue (and I can understand it) is that the font sizes are different between display mode and edit mode. Is that it?

File: addon/components/course-header.hbs (hunk 1 above)
Comment (stopfstedt) [reply addon/components/course-header.hbs]: handled.

**Comment by stopfstedt** [comment]
> Not sure about this solution, why would the title area only be bigger when editing, shouldn't it take up the same space as the full title did because it contains the same characters?

@jrjohnson would this be better then?

Selection_052

**Review by jrjohnson — APPROVED** [review]

**Comment by jrjohnson** [comment]
Egad, I really don't like that. I was worried about the year jumping around when editing that it might be distracting if it changed lines, but I didn't trust the CSS.  I just played with it a bunch and it works really well. I take back my objection, this is perfect.

**Pull request merged**